tasks/windows/collect-traceback-cinder-services.yaml

```diff
@@ -0,0 +1,50 @@
+    - name: Set additional variables for cinder-volume service
+      set_fact:
+        cinder_volume_service: "{{ item.name }}"
+        cinder_volume_binary: "{{ item.binary }}"
+        cinder_volume_config: "{{ item.config }}"
+      loop: "{{ services }}"
+      loop_control:
+        index_var: service_index
+      when: services[service_index].name == 'cinder-volume'
+
+    - name: "Check if {{ cinder_volume_service }} log exists"
+      win_stat:
+        path: "{{ win_dir.log }}\\cinder.log"
+        get_checksum: False
+      register: cinder_volume_log_stat
+
+    - name: "Run {{ cinder_volume_binary }} binary"
+      args:
+        executable: cmd
```

Comment:
Used as executable environment **cmd** so both **stdout** & **stderr** to be collected correctly. Using the default executable value powershell made not possible this log collection